commands/info.go

```diff
@@ -0,0 +1,76 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/ryanuber/columnize"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/gsctl/config"
+)
+
+var (
+	// InfoCommand is the "info" go command
+	InfoCommand = &cobra.Command{
+		Use:   "info",
+		Short: "Print some information",
+		Long:  `Prints information that might help you get out of trouble`,
+		Run:   printInfo,
+	}
+)
+
+// printInfo prints some information on the current user and configuration
+func printInfo(cmd *cobra.Command, args []string) {
+
+	output := []string{}
+
+	if config.Config.Organization == "" {
+		output = append(output, color.YellowString("Selected organization:")+"|"+"n/a")
+	} else {
+		output = append(output, color.YellowString("Selected organization:")+"|"+color.CyanString(config.Config.Organization))
+	}
+
+	if config.Config.Cluster == "" {
+		output = append(output, color.YellowString("Selected cluster:")+"|"+"n/a")
+	} else {
+		output = append(output, color.YellowString("Selected cluster:")+"|"+color.CyanString(config.Config.Cluster))
+	}
+
+	if config.Config.Email == "" {
+		output = append(output, color.YellowString("Email:")+"|"+"n/a")
+	} else {
+		output = append(output, color.YellowString("Email:")+"|"+color.CyanString(config.Config.Email))
+	}
+
+	if cmdVerbose {
+		if cmdToken != "" {
+			output = append(output, color.YellowString("Auth token:")+"|"+color.CyanString(cmdToken))
+		} else if config.Config.Token != "" {
+			output = append(output, color.YellowString("Auth token:")+"|"+color.CyanString(config.Config.Token))
+		} else {
+			output = append(output, color.YellowString("Auth token:")+"|n/a")
+		}
+	}
+
+	output = append(output, color.YellowString("%s version:", config.ProgramName)+"|"+config.Version)
+	output = append(output, color.YellowString("%s build:", config.ProgramName)+"|"+config.BuildDate)
+
+	output = append(output, color.YellowString("Config path:")+"|"+color.CyanString(config.ConfigFilePath))
```

Comment:
There is probably a reason but it looks like we are doing the table generation wrong. You may want to create an issue to improve this, if not already done.